http_stubs/templatetags/stub_tags.py

```diff
@@ -1,18 +1,52 @@
-from typing import Dict
+import json
+from html import unescape
+from json import JSONDecodeError
+from typing import AnyStr, Dict, List
+from urllib import parse
 
 from django import template
+from django.contrib.admin.helpers import Fieldset
+from django.template.defaultfilters import stringfilter
 
 Url = str
 
 register = template.Library()
 
 
 @register.simple_tag(takes_context=True, name='absolute')
-def get_absolute_url_tag(context: Dict, url: Url) -> Url:
+def get_absolute_url_tag(context: Dict, url: Url, fieldset: Fieldset) -> Url:
     """Tag that returns an absolute url.
 
     :param context: context of request
     :param url: relative url
+    :param fieldset: Fieldset of Django
```

Comment:
Such description is opaque to the reader. It is obvious that it would be a django fieldset, but reader would need information why is fieldset needed, when it should be passed. Correct description should be along these lines:

> fieldset that is used to get value of the `regex_path` field from the form

Reply:
Changed. Thanks!